/**:
  ros__parameters:
    update_rate: 10.0
    map_frame_id: "map"
    costmap_center__frame_id: "base_link"
    multi_layered_costmap:
      layers:
        - "lanelet_layer"
        # - "predicted_object_layer"
      lanelet_layer:
        type: "lanelet"
        map_topic: "/map/vector_map"
      # predicted_object_layer:
      #   type: "predicted_object"
      #   predicted_objects_topic: "/perception/object_recognition/objects"
      #   distance_threshold: 3.0
